includes/checkout/functions.php

```diff
@@ -608,3 +608,94 @@ function edd_purchase_form_validate_cc_exp_date( $exp_month, $exp_year ) {
 	return $expiration >= time();
 
 }
+
+/**
+ * Add payment SVG definitions to the EDD checkout.
+ * 
+ * @since 2.9.x
+ */
+function edd_payment_svg_icons() {
+	
+	if ( ! edd_is_checkout() ) {
+		return;
+	}
+
+	$payment_methods = edd_get_option( 'accepted_cards', array() );
+
+	if ( empty( $payment_methods ) ) {
+		return;
+	}
+
+	?>
+	<svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+		<defs>
+		<?php if ( array_key_exists( 'mastercard', $payment_methods ) ) : ?>
+			<symbol id="icon-mastercard" viewBox="0 0 50 32">
```

Comment:
Note: SVGs have been minified as much as possible, we ain't squeezing any more outta these 😉

Reply:
Once I get to it, this function will also be used to load the SVG icons into the payment settings page.